quickstart.py

```diff
@@ -261,14 +296,10 @@ def serversResponding(config):
       print "Timeout exceeded waiting for SymServer to start"
       return False
 
-    # Make sure we have a port value to contact SymServer on.
-    if 'configPath' in config:
-      SymServer.config.loadFile(config['configPath'])
-    elif 'configJSON' in config:
-      SymServer.config.loadJSON(config['configJSON'])
+    port = getSymServerConfig(config)['port']
```

Comment:
I also don't think that it is worth moving to INI just for comments. Especially given that JSON provides types and nesting.